p4sdk/images/Dockerfile

```diff
@@ -1,27 +1,35 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2024 Intel Corporation
-FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as builder
-ARG TARGETOS
-ARG TARGETARCH
+
+FROM registry.access.redhat.com/ubi9/ubi:9.3-1610
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --no-cache --virtual tar
 COPY . /src
 WORKDIR /src
-RUN mkdir -pv /opt/p4 && tar -zxvf p4.tar.gz -C /opt/p4 && rm -f p4.tar.gz
+RUN mkdir -pv /opt/ && tar -zxvf p4.tar.gz -C /opt/ && rm -f p4.tar.gz
 
-FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
-RUN apk add --no-cache --virtual openssl libressl-dev ncurses-libs libstdc++ gawk gettext
-COPY --from=builder /opt/p4 /opt/
 COPY p4 /root/linux_networking
```

Comment:
is the plan to replace the linux_networking pkg with the rh pkg?
i.e. just overwrite the files but keep the names?

Reply:
yes - will test this today as I'm looking to install the p4 rules. The existing artifacts will be replaces.